docs/UserGuide.md

```diff
@@ -99,15 +99,15 @@ Broadly speaking, ContaX consists of an *Address Book* for managing Contacts, an
 
 ### Viewing help : `help`
 
-Shows a message explaning how to access the help page.
+Displays a window with the commands' usage and a link to access the user guide.
 
 ![help message](images/helpMessage.png)
 
 Format: `help`
 
 ### Onboarding guide
 
-#### Prompt on first run
+Prompt on first run
```

Comment:
If you want to maintain this as a header, use Markdown Bold. It looks out of place without any formatting.

Reply:
FIxed